spring-webmvc/src/test/java/org/springframework/web/servlet/mvc/method/annotation/CrossOriginTests.java

```diff
@@ -345,6 +376,34 @@ public void baz() {
 
 	}
 
+	@Target({ElementType.METHOD, ElementType.TYPE})
+	@Retention(RetentionPolicy.RUNTIME)
+	@Documented
+	@CrossOrigin
+	private @interface CrossOriginMetaAnnotation {
```

Comment:
I'm being picky for a reason.... people often confuse "meta-annotation" and "composed annotation" and that makes it difficult to discuss the topic. So I'm a proponent for a ubiquitous language in this context. See the following for details:

https://github.com/spring-projects/spring-framework/wiki/Spring-Annotation-Programming-Model#terminology

With that in mind, something like `@ComposedCrossOrigin` or `@MyCrossOrigin` (or something directly related to the use case) would provide more clarity for the intent.